src/components/People.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react';
+import { Person } from '../types/Person';
+
+type Props = {
+  people: Person[];
+  onSelect: (person: Person | null) => void;
+};
+
+export const PeopleList: React.FC<Props> = ({ people, onSelect }) => {
+  const [hoveredPerson, setHoveredPerson] = useState<Person | null>(null);
+
+  const handlePersonClick = (person: Person) => {
+    onSelect(person);
+  };
+
+  return (
+    <div className="dropdown-content">
+      {people.length === 0 ? (
+        <div
+          className="
+            notification
+            is-danger
+            is-light
+            mt-3
+            is-align-self-flex-start
+          "
+          role="alert"
+          data-cy="no-suggestions-message"
+        >
+          <p className="has-text-danger">No matching suggestions</p>
+        </div>
+      ) : (
+        people.map((person) => (
+          <div
+            key={person.slug}
+            className={`dropdown-item ${person === hoveredPerson ? 'hovered' : ''}`}
```

Comment:
it is better to use libraries like `classnames` or `clsx` for conditional css classes